app/controllers/api/v1/incident_contacts_controller.rb

```diff
@@ -71,15 +59,19 @@ def incident_contact_params
     params.permit(:title, :incident_id, :contact_id)
   end
 
-  def distance_miles
```

Comment:
I see you removed these distance methods but I do believe you wanted to talk about how we are sending them to front end so that  works.